content/doc/advanced/customizing-gitea.en-us.md

```diff
@@ -17,6 +17,14 @@ menu:
 
 The main way to customize Gitea is by using the `custom` folder. This is the central place to override and configure features.
 
+Use `GITEA_CUSTOM` to define another folder instead. For example:
+
+```
+GITEA_CUSTOM=/srv/gitea/mycustom ./gitea web
+```
+
+**Note** that by default Gitea will use the `custom` folder in the current working directory.
```

Comment:
You're right, I didn't be aware that GITEA_WORK_DIR variable exists.
That's why it should be better to create a specific subsection in this page to explain how GITEA_WORK_DIR and GITEA_CUSTOM works.
Or, in another case: create a specific page to list all available environment variable for GITEA?
What's your opinion about that?